<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QWidget" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="batteryBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>118</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>12:00 AM</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDate">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>26 Apr</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="insulinRemainingBar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>118</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>-1</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="batteryLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
  </widget>
  <widget class="QLabel" name="insulinRemaining">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>30</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>300U</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="graph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>531</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>Graph Placeholder</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentBldGlu">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>6.3</string>
   </property>
  </widget>
  <widget class="QLabel" name="metricBld">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>mmol/L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeWindow">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>180</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="hours">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>HRS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="optionsButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>OPTIONS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>INSULIN ON BOARD:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0 u</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bolusButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>BOLUS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>20</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Tandem LOGO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="powerButton_3">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>150</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>POWER BUTTON</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chargingButton">
   <property name="geometry">
    <rect>
     <x>939</x>
     <y>110</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Charging</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refillButton">
   <property name="geometry">
    <rect>
     <x>939</x>
     <y>150</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Refill</string>
   </property>
  </widget>
  <widget class="QWidget" name="powerOffOverlay" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>911</width>
     <height>291</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="logoButton_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>20</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Tandem LOGO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>140</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>POWER BUTTON</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Pump is Off 
Hold Power Button to Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>591</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button1">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 80px;        /* Half of width/height */
    background-color: #444;
    color: white;
    font-size: 24px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #666;
}
QPushButton:checked {
    background-color: #2ecc71;  /* Green */
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button2">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 80px;        /* Half of width/height */
    background-color: #444;
    color: white;
    font-size: 24px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #666;
}
QPushButton:checked {
    background-color: #2ecc71;  /* Green */
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button3">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>160</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 80px;        /* Half of width/height */
    background-color: #444;
    color: white;
    font-size: 24px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #666;
}
QPushButton:checked {
    background-color: #2ecc71;  /* Green */
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="Disconnect">
   <property name="geometry">
    <rect>
     <x>939</x>
     <y>231</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Simulate 
Disconnection</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="activeUserTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Basal Rate</string>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Carb Ratio</string>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Correction Factor</string>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Glucose Target</string>
    </property>
    <property name="font">
     <font>
      <pointsize>6</pointsize>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
